<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MODISView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1350</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Image</string>
     </attribute>
     <widget class="imwidget" name="image_widget" native="true">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>10</y>
        <width>1320</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
     <widget class="myplotwidget" name="plot_widget0" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>670</y>
        <width>1320</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>TimeSeries</string>
     </attribute>
     <widget class="myplotwidget" name="plot_widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>670</y>
        <width>941</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="timeseriesResTE">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>20</y>
        <width>351</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>170</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="dayButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Day</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="nightButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Night</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>50</y>
      <width>101</width>
      <height>115</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="month_CB">
       <item>
        <property name="text">
         <string>Jan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Apr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jul</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sep</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oct</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nov</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dec</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yearLE"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="landtempRB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Land Temp.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NDVI</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="flabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current File : </string>
    </property>
   </widget>
   <widget class="QLabel" name="fnameLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>550</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>260</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Location</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>32</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>87</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lonLE">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>56</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="latLE">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>111</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
    <addaction name="actionNew"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../PyMODIS/Open.bmp</normaloff>../PyMODIS/Open.bmp</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit MODISView</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>imwidget</class>
   <extends>QWidget</extends>
   <header>imwidget.h</header>
  </customwidget>
  <customwidget>
   <class>myplotwidget</class>
   <extends>QWidget</extends>
   <header>myplotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../PyMODIS/resources.qrc"/>
 </resources>
 <connections/>
</ui>
